tests/ResourceGeneratorTest.php

```diff
@@ -152,6 +152,27 @@ public function it_accepts_attributes_which_are_saved_to_the_model()
     }
 
     /** @test */
+    public function it_accepts_float_attributes_which_are_saved_to_the_model()
+    {
+        $model = 'Louse';
+
+        // When I create a new model called 'Animal'.
+        $this->runArtisanCommand(ResourceMakeCommand::class, [
+            'name' => $model,
+            'attributes' => 'eyes:float',
```

Comment:
Sorry for the easter egg
"which animal has more than three and less than four eyes"
It is **PI**olho
**PI**olho is a Louse and eyes is a "olho" in portuguese.
And PI equals 3.14....